xdmod-appkernels.spec.in

```diff
@@ -51,9 +51,14 @@ rm -rf $RPM_BUILD_ROOT
 %{_libdir}/xdmod/
 %{_datadir}/xdmod/
 %{_docdir}/%{name}-%{version}__PRERELEASE__/
+%{_docdir}/%{name}-%{version}/
+%{_sysconfdir}/xdmod/modules.d/appkernels.json
 
 %config(noreplace) %{_sysconfdir}/xdmod/*.d/appkernels.ini
-%config(noreplace) %{_sysconfdir}/xdmod/*.d/appkernels.json
+%config(noreplace) %{_sysconfdir}/xdmod/assets.d/appkernels*.json
+%config(noreplace) %{_sysconfdir}/xdmod/internal_dashboard.d/appkernels*.json
+%config(noreplace) %{_sysconfdir}/xdmod/roles.d/appkernels*.json
+%config(noreplace) %{_sysconfdir}/xdmod/setup.d/appkernels*.json
```

Comment:
Why is this line being replaced with four lines if nothing changed with these files?

Reply:
If memory serves, @ryanrath wanted to take into account some sort of versioning or other appkernels files, but it looks like the same thing could be accomplished simply by changing `appkernels.json` to `appkernels*.json`

Reply:
So leaving out the fact that I forgot to include the modules.d/appkernels.json file. The reason for the change is so that the modules.d/appkernels.json is included but not flagged as config(noreplace) while appkernels*.json in the other folders are flagged as config(noreplace).